libs/payments/ui/src/lib/actions/checkoutCartWithStripe.ts

```diff
@@ -20,4 +21,6 @@ export const checkoutCartWithStripe = async (
       paymentMethodId,
     })
   );
+
+  revalidatePath('/');
```

Comment:
Imo this should be outside of the action itself. Action should only have the responsibility of dispatching the call to the server.